docs/production-deployment.rst

```diff
@@ -151,34 +148,54 @@ Then, it advised to use the `--dry-run --debug` options before deploying with `h
 Installation
 ~~~~~~~~~~~~
 
-To install the whole system execute::
+To install the whole system assuming you have a configuration file ``my-deployment.yaml`` in the current
+directory::
 
-    cd deployment
-    helm install entityservice --name="anonlink" --values ``my-deployment.yaml``
+    $ helm upgrade --install anonlink data61/entity-service -f anonlink.yaml
```

Comment:
shouldn't that be `my-deployment.yaml` instead of `anonlink.yaml`? Or is there some strange magic involved?

Reply:
No magic. Good catch